lib/Marlin/Marlin/src/core/macros.h

```diff
@@ -733,3 +736,8 @@
 #define __MAPLIST() _MAPLIST
 
 #define MAPLIST(OP,V...) EVAL(_MAPLIST(OP,V))
+
+// Temperature Sensor Config
+#define _HAS_E_TEMP(N) || (TEMP_SENSOR_##N != 0)
+#define HAS_E_TEMP_SENSOR (0 REPEAT(EXTRUDERS, _HAS_E_TEMP))
+#define TEMP_SENSOR_IS_MAX_TC(T) (TEMP_SENSOR_##T == -5 || TEMP_SENSOR_##T == -3 || TEMP_SENSOR_##T == -2)
```

Comment:
Yuri explained, that it comes from later marlin 2.1.x branch than my update marlin was based on.

Reply:
I feel like this sort of macro should have been in one of the Conditionals_*, but yeah, it's in upstream.